cockatrice/src/userlist.cpp

```diff
@@ -307,7 +307,7 @@ bool UserListTWI::operator<(const QTreeWidgetItem &other) const
 
     // Sort by user level
     if (data(0, Qt::UserRole) != other.data(0, Qt::UserRole))
-        return data(0, Qt::UserRole).toInt() > other.data(0, Qt::UserRole).toInt();
+        return (data(0, Qt::UserRole).toInt() & 15) > (other.data(0, Qt::UserRole).toInt() & 15);
```

Comment:
Do we want to &15 instead of & role name?

Reply:
There is no role name for 15, it's all the roles below judge or'd together.

Reply:
-facepalm-